torch/csrc/jit/codegen/cuda/test/test_gpu.cpp

```diff
@@ -3534,6 +3534,7 @@ TEST_F(NVFuserTest, FusionRootMappingReductionDependency6_CUDA_CUDA) {
       {true, true});
 }
 
+#if 0
```

Comment:
CC @naoyam for disabled tests. This is likely fine and comes from the changes in `root_domain_map`. I think we should probably change the test to see if it fails in lowering as these cases (maybe not all, haven't looked through them carefully) seem like they should still be caught as not supported.

Reply:
Removed in #1941